src/status_im2/contexts/profile/settings/header/avatar.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im2.contexts.profile.settings.header.avatar
+  (:require [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.reanimated :as reanimated]
+            [status-im2.contexts.profile.settings.header.style :as style]))
+
+(defonce ^:const scroll-animation-input-range [0 50])
+
+(def header-extrapolation-option
+  {:extrapolateLeft  "clamp"
+   :extrapolateRight "clamp"})
+
+(defn f-avatar
+  [{:keys [scroll-y full-name online? profile-picture customization-color]}]
+  (let [image-scale-animation       (reanimated/interpolate scroll-y
+                                                            scroll-animation-input-range
+                                                            [1 0.5]
+                                                            header-extrapolation-option)
+        image-top-margin-animation  (reanimated/interpolate scroll-y
+                                                            scroll-animation-input-range
+                                                            [0 20]
+                                                            header-extrapolation-option)
+        image-side-margin-animation (reanimated/interpolate scroll-y
+                                                            scroll-animation-input-range
+                                                            [0 -20]
+                                                            header-extrapolation-option)
+        theme                       (quo.theme/get-theme)]
```

Comment:
to match the border with background, i need to know the exact theme behind the settings screen

Reply:
can you not use the context provider here?
i.e `(quo.theme/use-theme-value)`

Reply:
@J-Son89 how can i get selected theme then?  i'm using `:options   options/transparent-screen-options` for this screen and `quo.theme/with-theme` always return `dark`

Reply:
yes please let me know if it has any better alternative , but why we don't want to use `(quo.theme/use-theme-value)` ?

Reply:
`quo.theme/get-theme` returns what is sometimes called the `user theme` 

https://github.com/status-im/status-mobile/blob/c1dcd7a76404bed90a5c9b6913a5ef8016fc1382/src/status_im2/navigation/view.cljs#L64

`quo.theme/use-theme-value` returns the current theme, but for a specific depth of the component tree by using the React context `quo.theme/theme-context`.

Both return a "current theme", but their semantics are different.

I see in Figma some components in the settings screen are using a light theme (`Avatar` and `Floating Shell Button`), while the vast majority are using dark theme (even when the OS theme is light, similar to the Activity Center). https://www.figma.com/file/JlpPhJp0SMnEyBQy4nOZHo/Settings-for-Mobile?type=design&node-id=5345-441704&mode=design&t=p7BEbEetL7j9st0U-0

If Figma is correct and the avatar component must follow the OS theme even when the settings screen is dark only, then `quo.theme/get-theme` is the only way and the original code is correct. Maybe we should confirm with a designer if the avatar component shouldn't also always be in dark theme in the dark-only settings screen.

Also, the naming is confusing. To me, `quo.theme/theme-state` atom should be renamed to something like `quo.theme/user-theme-state` and `quo.theme/get-theme` to `quo.theme/get-user-theme`. Maybe not even renaming would make things clear...

Reply:
@ilmotta thanks for explanation, i need to mention i used user's theme to make border color around avatar like background color, i didn't use it in the avatar.
![285901630-f7d9f130-f18d-4409-b764-f37c67244782](https://github.com/status-im/status-mobile/assets/71308738/f1c0897c-193d-4201-a883-d4d147e86400)

Reply:
Yeah that can be a mistake but for this screen doesn't matter as Pedro said this screen should be theme agnostic. i just needed to somehow simulate that curvy shape(transparent space between avatar and top header), so i gave it a outer border with a color that follow user theme. @ilmotta